commands.go

```diff
@@ -140,7 +140,7 @@ type Cmdable interface {
 	BitPos(ctx context.Context, key string, bit int64, pos ...int64) *IntCmd
 	BitField(ctx context.Context, key string, args ...interface{}) *IntSliceCmd
 
-	Scan(ctx context.Context, cursor uint64, match string, count int64) *ScanCmd
+	Scan(ctx context.Context, cursor uint64, match string, keyType string, count int64) *ScanCmd
```

Comment:
It looks like in Redis the `type` is the last arg. Besides this is a breaking change.

Can we instead add `ScanType(ctx, cursor, match, count, keyType)`? That is backwards compatible and more explicit.

Reply:
That would be a better solution